huntsman/utils/hub_utils.py

```diff
@@ -0,0 +1,219 @@
+#A Python interface for the acroname USB Hubs - tested and functional
+
+#Please note that the Yaml config file that monique is working on is not yet finalised and is not accessable in POCS yet
+# so not sure what the best approach is here to retrieve info from the file - this can be updated once this is finished 
+
+import brainstem
+import time
+import yaml
+from brainstem.result import Result
+from time import sleep
+
+
+def connect_to_hub(hub_serial, verbose = False, test = True):
+    
+    """
+    Function: Used to connect to the hub and create stem
+    
+    Inputs:
+        
+        hub_serial: Serial number of the hub
+        
+        test: If true, will flash user LED's to verify that a connection has 
+        been made 
+        
+    Outputs: 
+        
+        stem: Connection to hub    
+    
+    """
+    
+    print('Connecting to hub', hub_serial)
```

Comment:
We might not always want to `print` so we need to control printing via a `verbose` flag. See an example [here](https://github.com/panoptes/POCS/blob/develop/pocs/utils/images/__init__.py#L194).

Reply:
I see you have `verbose` as an option up above. For odd reasons I can explain in person it's better to use the `kwargs` as in example in comment above. Basically this allows `verbose` to pass through to other util functions without being "consumed" by this one.